src/agent/Core/SpawningKit/Spawner.h

```diff
@@ -871,8 +872,42 @@ class Spawner {
 		}
 	}
 
+	void enterLveJail(const struct passwd * pw) {
+		if (!pw)
+			return;
+
+		string lve_init_err;
+		adhoc_lve::LibLve& liblve = adhoc_lve::LveInitSignleton::getInstance(&lve_init_err);
+		if (liblve.is_error())
+		{
```

Comment:
Please also ensure here that the `{` comes directly after the `if` statement, like how it is done for functions:

    if (liblve.is_error()) {